runtime/compiler/env/PersistentCHTable.cpp

```diff
@@ -500,7 +500,7 @@ TR_PersistentCHTable::findSingleAbstractImplementer(
    {
    if (comp->getOption(TR_DisableCHOpts))
       return 0;
-   TR_PersistentClassInfo * classInfo = findClassInfoAfterLocking(thisClass, comp);
+   TR_PersistentClassInfo * classInfo = findClassInfoAfterLocking(thisClass, comp, true);
```

Comment:
It would be more conventional to introduce a default parameter, and pass it through here. However, it looks to me like you hit all of the callers except for [one in invariant argument preexistence](https://github.com/eclipse/omr/blob/02bfd61c0fd38b2b641828e879f0bc7ad062db8d/compiler/optimizer/LocalOpts.cpp#L7497), which I believe (currently) doesn't run for AOT anyway.

Reply:
I sort of followed what happens in findSingleInterfaceImplementer, which does something similar :P